.github/settings.yml

```diff
@@ -0,0 +1,34 @@
+# Collaborators: give specific users access to this repository.
+# See https://docs.github.com/en/rest/reference/repos#add-a-repository-collaborator for available options
+collaborators:
+  # Maintainers
+  - username: mmlb
+    permission: maintain
+  - username: micahhausler
+    permission: maintain
+  # Approvers
+  - username: jacobweinstock
+    permission: push
+  - username: tstromberg
+    permission: push
+  - username: displague
+    permission: push
+  - username: nshalman
+    permission: push
+  # Reviewers
+  - username: tobert
+    permission: triage
+  - username: stephen-fox
+    permission: triage
+  - username: chrisdoherty4
+    permission: triage
+  - username: detiber
+    permission: triage
```

Comment:
What do you think about in this form?
```yaml
collaborators:
  # Maintainers
  - { permission: maintain, username: micahhausler }
  - { permission: maintain, username: mmlb }
  # Approvers
  - { permission: push, username: displague }
  - { permission: push, username: jacobweinstock }
  - { permission: push, username: nshalman }
  - { permission: push, username: tstromberg }
  # Reviewers
  - { permission: triage, username: chrisdoherty4 }
  - { permission: triage, username: detiber }
  - { permission: triage, username: stephen-fox }
  - { permission: triage, username: tobert }
```

I like it because the groups are more obvious this way.

Reply:
There are some readability perks, but I think they are marginal (🥁 ).

I'm fine with the original (boring) yaml list, fwiw.

Reply:
no one ever got fired for picking vanilla :) Lets just go with the original.